apps/dashboard/src/app/retrieval-agent/agent-dashboard/agent-dashboard.component.scss

```diff
@@ -33,8 +33,14 @@
       display: flex;
       flex-direction: column;
       gap: rhythm(3);
-      height: fit-content;
       padding: rhythm(8);
+
+      .column-section {
```

Comment:
It seems that "column sections" might overlap each other when there are many nodes. I don't know if it's a problem.

Reply:
definitely a problem yes, good catch !